examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -202,9 +202,11 @@ private static void performance_calibration_pricing() {
         .build();
 
     // calculate the results
-    CalculationRunner runner = CalculationRunnerFactory.ofSingleThreaded()
-        .createWithMarketDataBuilder(trades, columns, rules, marketDataFactory(), marketDataConfig);
-    return Pair.of(trades, runner.calculateSingleScenario(marketSnapshot));
+    return CalculationRunnerFactory.multiThreaded(factory -> {
```

Comment:
It feels to me like this call tries to do too much. Something like the following would be more explicit and potentially clearer:

```
try (CalculationRunner runner = CalculationRunnerFactory.createMultiThreaded()) {
  runner.calculateSingleScenario(...)
}
```

This implies the `CalculationRunner` itself being the thing that's `AutoCloseable`, which is what I would more naturally expect than the factory.